lib/internal/bootstrap_node.js

```diff
@@ -4,11 +4,14 @@
 // responsible for bootstrapping the node.js core. As special caution is given
 // to the performance of the startup process, many dependencies are invoked
 // lazily.
+//
+// Before this file is run, lib/internal/native_module.js gets run first
+// to bootstrap the internal binding and module loaders, including
+// process.binding(), NativeModule and internalBinding().
 
 'use strict';
 
-(function(process) {
-  let internalBinding;
+(function bootstrapNodeJSCore(process, NativeModule) {
```

Comment:
I’m wondering whether this function name showing up is more helpful or more confusing when showing up ni embedder code…